app/src/types/User.ts

```diff
@@ -0,0 +1,13 @@
+import { IStamps } from '../interfaces/IStamps';
+
+export type User = {
+  userId?: string; // Primary Key
```

Comment:
I guess while we're at this, we should put in the Primary Key in-line comment on all of our type models as if you were just looking at this type, it wouldn't always be immediately clear which attribute/property was behaving as a primary key.